src/status_im2/contexts/wallet/send/select_asset/view.cljs

```diff
@@ -18,10 +18,13 @@
 (defn- asset-component
   []
   (fn [token _ _ _]
-    (let [on-press                #(js/alert "Not implemented yet")
+    (let [on-press
+          #(if (= (:symbol token) "ETH")
```

Comment:
is there any reason we need to do this for the moment? does the app crash otherwise? if it's to prevent the app crashing perhaps we should just handle that instead of this ? np either way for me as I realise the updates are coming v soon!